File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_get_temp_session_form.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusGetTempSessionForm:
    """Temporary session request form"""
    provider: str = None
    name: str = None
    scene: str = None
    bucket: BucketObject = None
    method: str = None
    expiration_seconds: int = None

File: sdkwork-backend-sdk-go/types/plus_get_temp_session_form.go
package types

// Temporary session request form
type PlusGetTempSessionForm struct {
	Provider string `json:"provider"`
	Name string `json:"name"`
	Scene string `json:"scene"`
	Bucket BucketObject `json:"bucket"`
	Method string `json:"method"`
	ExpirationSeconds int `json:"expirationSeconds"`
}
